include/cudaq/Optimizer/Dialect/Quake/QuakeOps.td

```diff
@@ -1397,4 +1397,89 @@ def CustomUnitarySymbolOp :
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// Quantum states
+//===----------------------------------------------------------------------===//
+
+def quake_CreateStateOp : QuakeOp<"create_state", [Pure] > {
+  let summary = "Create state from data";
+  let description = [{
+    This operation takes a pointer to state data and creates a quantum state.
```

Comment:
Maybe add some more words about what "state data" has to be?

Reply:
done